internal/api/v1/service/delete.go

```diff
@@ -101,7 +102,7 @@ func Delete(c *gin.Context) apierror.APIErrors {
 			return apierror.InternalError(err)
 		}
 
-		logger.Info(fmt.Sprintf("configurationSecrets found %+v\n", serviceConfigurations))
+		logger.Info("configurations", "service", service.Meta.Name, "configurations", serviceConfigurations)
```

Comment:
Since the `serviceConfigurations` are `v1.Secrets` the logging will be too verbose/annoying.
I would print only the len of them, and then in the loop we could print what we need (name and namespace?).

```go
logger.Info(fmt.Sprintf("found %d configurations", len(serviceConfigurations)), "service", service.Meta.Name)

for _, secret := range serviceConfigurations {
	logger.Info(fmt.Sprintf("configuration secret for service '%s'", service.Meta.Name),
		"name", secret.Name,
		"namespace", secret.Namespace,
	)
}
```